pypeit/core/skysub.py

```diff
@@ -314,10 +314,10 @@ def optimal_bkpts(bkpts_optimal, bsp_min, piximg, sampmask, samp_frac=0.80,
            adopt a uniform break point spacing, rather adopting the optimally spaced breakpoints.
         skyimage: ndarray, shape = (nspec, nspat), default = None
            Sky model image used only for QA.
-        mincol: float, default = None
-           Minimum spatial column used for local sky subtraction fitting. Only used for title of QA plot.
-        maxcol: float, defualt = None
-           Maximum spatial column used for local sky subtraction fitting. Only used for title of QA plot.
+        min_spat: float, default = None
```

Comment:
I don't include defaults in the description.  It's in the function call in the sphinx docs, and it keeps you from having to change this if the default changes.